packages/plugins/components/src/NavigationDropdown/NavigationDropdown.tsx

```diff
@@ -20,39 +27,48 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   colorInherit: {
     color: 'inherit',
-    fontFamily: headerFontFamily,
+    fontFamily: ({ headerFontFamily }) => headerFontFamily,
     fontWeight: 600,
     lineHeight: 1.75,
   },
 }));
 
-interface NavigationDropdownProps {
+export type MenuItemProps = typeof MenuItem;
+
+export interface Config extends StyledComponentProps {
+  headerFontFamily: string;
+  Select?: Omit<SelectProps, 'open' | 'onClose' | 'onOpen' | 'value'>;
+  MenuProps?: Pick<SelectProps, 'MenuProps'>;
+  DefaultMenuItem?: MenuItemProps;
+}
+
+export interface NavigationDropdownProps {
+  baseUrl: string;
   items: FlyteNavItem[]; // all other navigation items
-  console?: string; // name for default navigation, if not provided "Console" is used.
+  config: Config;
 }
 
 /** Renders the default content for the app bar, which is the logo and help links */
 export const NavigationDropdown = (props: NavigationDropdownProps) => {
-  // Flyte Console list item - always there ans is first in the list
-  const ConsoleItem: FlyteNavItem = React.useMemo(() => {
-    return {
-      title: props.console ?? 'Console',
-      url: makeRoute('/'),
-    };
-  }, [props.console]);
+  const { baseUrl, items: menuItems, config } = props;
 
-  const [selectedPage, setSelectedPage] = React.useState<string>(ConsoleItem.title);
+  const basePathName = React.useMemo(() => getBasePathName(), []);
```

Comment:
nit: update useState and useMemo imports
`import {useMemo, useState} from 'react';`